kubernetes/client/api_client.py

```diff
@@ -74,7 +74,10 @@ def __init__(self, configuration=None, header_name=None, header_value=None,
         self.cookie = cookie
         # Set default User-Agent.
         self.user_agent = 'Swagger-Codegen/10.0.0-snapshot/python'
-    
+
+        # prefered content-type if multiple options are allowed (eg. patch command)
```

Comment:
1) if multiple options are allowed and 2) if `application/json` is not allowed?

Reply:
I've removed prefered_content_type.